Superposition/ReferenceImplementation.qs

```diff
@@ -262,8 +302,55 @@ namespace Quantum.Kata.Superposition {
     }
     
     
-    // Task 12. W state on 2^k qubits
-    // Input: N = 2^k qubits in |0...0⟩ state.
+    // ------------------------------------------------------
+    // Task 13*. Superposition of four bit strings
+    // Inputs:
+    //      1) N qubits in |0...0⟩ state
+    //      2) four bit string represented as Bool[][] bits
+    //         bits is an array of size 4 x N which describes the bit strings as follows:
+    //         bits[i] describes the i-th bit string and has N elements;
+    //         bit values false and true correspond to |0⟩ and |1⟩ states.
+    //
+    // Goal: create an equal superposition of the four basis states given by the bit strings.
+    //
+    // Example: for N = 2 and bits = [[false, true], [true, false], [false, false], [true, true]]
```

Comment:
Given that there are only four computational basis states for two qubits, perhaps a three-qubit example would make it more clear what this task covers?